src/api-service/__app__/onefuzzlib/pools.py

```diff
@@ -889,10 +885,6 @@ def cleanup_nodes(self) -> bool:
 
         nodes_to_reset = [x for x in nodes if x.state in NodeState.ready_for_reset()]
```

Comment:
i suggest rename `nodes` here to help the readability

Reply:
Not sure what you mean.  Can you provide more context?

Reply:
Handled after offline discussion